src/effects/builtin/echoeffect.cpp

```diff
@@ -82,7 +82,8 @@ EffectManifestPointer EchoEffect::getManifest() {
     send->setName(QObject::tr("Send"));
     send->setShortName(QObject::tr("Send"));
     send->setDescription(QObject::tr(
-        "How much of the signal to send into the delay buffer"));
+        "How much of the signal to send into the delay buffer\n"
+        "When the effect unit is in D+W mode, keep this turned up all the way"));
```

Comment:
Have you actually tried this? The Send parameter acts as a volume knob for the whole chain in D+W mode.

Reply:
Yes, that is the essence of a send knob in  a return Mixxx.

Reply:
Right, so the only use case for the Send parameter in this mode is to smoothly fade the effect out. It cannot be used to adjust the level of the Echo relative to the rest of the chain as it behaves in D/W mode. I see now that the wording of this tooltip may imply that the user should never use the Send parameter in D+W mode even in the case of smoothly fading out the effect, so I'll clarify that: "When the effect unit is in D+W mode, keep this turned up all the way unless fading the effect out"

Reply:
Isn't this the original use case for this?
How about: "Note: In D+W mode the output of the delay buffer is the only input of the following effect."

Reply:
Maybe it would be better to fully explain how the effect behaves in each mode:
"In D/W mode, the dry signal is passed through.
In D+W mode, only the echoed signal is output."